packages/wonder-blocks-popover/src/components/__tests__/popover-dialog.test.js

```diff
@@ -13,33 +12,37 @@ jest.mock("@khanacademy/wonder-blocks-tooltip");
 describe("PopoverDialog", () => {
     it("should update the tail color to match the content's color", () => {
         // Arrange
-        const onUpdateMock = jest.fn();
+        const tooltipTailSpy = jest.spyOn(Tooltip, "TooltipTail");
```

Comment:
@somewhatabstract that's a fair point! I added back the test and mocked TooltipTail as you suggested. I felt that this assertion would be better as with the HTML one is hard to understand/read what do we need to expect from the test + it would easily break by a minor SVG change.